modules/javafx.base/src/test/java/test/util/memory/JMemoryBuddy.java

```diff
@@ -121,7 +120,7 @@ private static int checkCollectable(int stepsLeft, WeakReference weakReference)
         }
 
         if (weakReference.get() == null && counter < steps / 3) {
-            int percentageUsed = (int) ((steps - counter) / steps * 100);
+            int percentageUsed = (steps - counter) / steps * 100;
```

Comment:
I suspect this will not produce a desired result.
Perhaps it should be 
(int)(100.0 * (steps - counter) / steps);

Reply:
Removing the `(int)` cast is a behavior-neutral change. What you are pointing out does look a bug, but it is unrelated to this cleanup, so should be filed and fixed separately.

Reply:
Yeah, this looks like a bug, as `(steps - counter) / steps` which are all `int`s will likely always result in `0`.  The cast removal here however does not alter the calculation, but does point to an error in the author's thought process.  The result is however only used in the log message to inform of a problem, and so the logic is not otherwise affected.

I can file a separate issue for this, unless we can agree to fix it as part of this clean-up.

Reply:
cc: @FlorianKirmaier